web/server/codechecker_server/cmd/server.py

```diff
@@ -394,6 +394,10 @@ def arg_match(options):
             args.sqlite = args.sqlite.replace('<CONFIG_DIRECTORY>',
                                               args.config_directory)
 
+        # Convert relative sqlite file path to absolute.
+        if 'sqlite' in args and not os.path.isabs(args.sqlite):
```

Comment:
Do we have to check if the path is absolute? `os.path.abspath()` does returns its parameter itself if that's absolute.